Library/Formula/sfml.rb

```diff
@@ -27,9 +17,45 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
```

Comment:
We need patches to go upstream. Otherwise we keep the formula as-was, unfortunately.